src/code/z_map_exp.c

```diff
@@ -364,7 +364,7 @@ void Minimap_Draw(PlayState* play) {
 
     OPEN_DISPS(play->state.gfxCtx, "../z_map_exp.c", 626);
 
-    if (play->pauseCtx.state < 4) {
+    if (play->pauseCtx.state < 3) {
```

Comment:
What is the rationale for this change here?

Reply:
This actually fixes the pause crash that happens on emulators when pausing with the minimap open!

Reply:
Yes, it fixes a bug that causes a crash, so I don't think we need an ifdef here.